omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -367,6 +367,21 @@
                         SupportsInformationals="yes"
                         SupportsWarnings="yes"/>
             </Component>
+            <!-- Windows service declaration for Security agent -->
```

Comment:
@derekwbrown Could you validate this section ?

Reply:
Discussed with @derekwbrown and decided for now we are going to omit it from the MSI.